README.md

```diff
@@ -1,3 +1,8 @@
 # banyan shared repos
 
 you need to put the codecov badge here
+
+
+# Testing 
+
+- Remember to set ENV variables for testing
```

Comment:
make this a TODO so it gets picked up by the CI hook thing

Reply:
Done